Modules/_sqlite/connection.c

```diff
@@ -23,7 +23,6 @@
 
 #include "cache.h"
 #include "module.h"
-#include "structmember.h"         // PyMemberDef
```

Comment:
If PyMemberDef is only used in the C file, IMO it's better to not add it to the H file.

Reply:
Good point. I'll fix.

Reply:
Put it back to where it was in commit c9357792be

Reply:
Ah, I need to fix this again. PyMemberDef is used in a couple of files... I'll add this include to each file where it's used, so it's more explicit.

Reply:
… and `statement.c`

Reply:
Sorry, I had to rebase and force-push. This is all fixed in 6be86c9882a33f27ac25d50880edf6035ab230b1, f184612542abe9a1d03a4731a30059c7ac16e517, 926e2386edb3482e0879a9489ef1425d8692e9a0, and b98761c865a729849e13dbd134d0b4380ce0af94. PTAL and sorry 'bout the force push!